### DISCUSSION TITLE
Vs code not tracking migrated properties

### DISCUSSION THREAD

**Discussion by scerelli** [prisma/prisma#6692] [Q&A] [answered]
Hello I just started using Prisma and I have some issues making it work with vsCode. 
I installed the Prisma extension in Code but basically it seems to have some problem catching up with the evolving of my schema. 

what i did:
I created a schema with certain properties, than I added few other and ran a migration, what happen is that VSCode can recognize only the properties of the initial schema, so the types used seems to not respect the real migrated schema.

my initial schema:
```
model Shop {
  id          Int       @id @default(autoincrement())
  name        String    @unique @db.VarChar(255)
  scope       String    @db.VarChar(255)
}

```

i added a property
```
model Shop {
  id          Int       @id @default(autoincrement())
  name        String    @unique @db.VarChar(255)
  scope       String    @db.VarChar(255)
  accessToken String    @db.VarChar(255)
}
```

```
await prisma.shop.create({
    data: {
      name: shop,
      scope,
      accessToken, // this one returns an error
    }
  })
``` 
Any idea?

**Comment by janpio** [comment, marked as answer]
Did you run `prisma generate` in the meantime to generate a new Prisma Client, based on the new, updated schema?
(Some Migration commands to that automatically in the background)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
